cloud-sql/postgres/sqlalchemy/requirements.txt

```diff
@@ -2,3 +2,4 @@ Flask==1.1.2
 SQLAlchemy==1.3.17
 pg8000==1.13.2; python_version > '3.0'
 pg8000==1.12.5; python_version < '3.0'
```

Comment:
Since the sample uses Python 3+ syntax that doesn't work with 2.7, probably best to just pin to pg8000==1.13.2 to avoid creating the expectation that it will work with Python versions < 3.0.

Reply:
Done!